src/components/YourCompany/NameTranslations/ListNameTranslation.vue

```diff
@@ -18,11 +18,14 @@
 
         <br v-if="translation.action">
         <v-chip v-if="translation.action === ActionTypes.ADDED"
-          x-small label color="#1669BB" text-color="white">ADDED</v-chip>
+          x-small label color="$app-blue" text-color="white">ADDED</v-chip>
```

Comment:
Does this work?

AFAIK, this is a Vuetify element and only knows about Vuetify theme colors.

If this isn't working, use `primary` here, which is configured the same as $app-blue here:
https://github.com/bcgov/business-edit-ui/blob/79f008d48abd514d3901a86c5116676e76b86e51/src/main.ts#L71

Reply:
Gotcha